components/Shared/Input/Funds.jsx

```diff
@@ -212,10 +194,10 @@ const Funds = forwardRef(
             >
               {'\u003D'}
             </Box>
-            <DenomTag top='30px' left='30px'>
+            <DenomTag top={4} left={5}>
               FIL
             </DenomTag>
-            <DenomTag top='110px' left='30px'>
+            <DenomTag top='105px' left={5}>
```

Comment:
ugh, tried to use themes as much as possible, but i at least stuck to the rule we decided on... _inside_ the shared directory, you can deviate from theme